rewrite-kotlin/src/test/java/org/openrewrite/kotlin/cleanup/MoveLambdaArgumentParenthesesTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.kotlin.cleanup;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+import org.openrewrite.test.TypeValidation;
+
+import static org.openrewrite.kotlin.Assertions.kotlin;
+
+class MoveLambdaArgumentParenthesesTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new MoveLambdaArgumentParentheses())
+                .typeValidationOptions(TypeValidation.none())
+                .expectedCyclesThatMakeChanges(1)
+                .cycles(1);
```

Comment:
Let's try to see if we can make the recipe work without these escape hatches; each of these disable a check the proper functioning of the recipe, which could hurt the ability to chain recipes back to back.

For the cycles it helps to make no change at all if the desired state is already reached, either before starting, or after a second invocation.

For the type validation you might want to look at using template placeholders and arguments, as opposed to a string-ified template.

```suggestion
        spec.recipe(new MoveLambdaArgumentParentheses());
```

Reply:
Thank you for review pull request. 
I edited my test cases to work without those escape hatches; however, one test case still needs skip-type validation. Until now, I haven't found a way to pass the validation (without escape hatches). What should I do about that?

Reply:
Thanks for the changes! I'll see if I can help clear out that last hurdle before we merge.

Reply:
Please let me know if you think there is anything I can do to find a different way. Thanks.